### PR TITLE
[User] List Follow and Unfollow

### PR DESCRIPTION
**Pull request by cntt11b1**

### CODE REVIEW

File: app/views/users/_show_follow.html.erb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <div class="nav-tabs-custom">
+          <div class="feed_title">
+            <h2><%= t "followers" %></h2>
+          </div>
+          <div class="tab-content" id="show_follow">
+            <div class="table-responsive">
+               <table class="table table-striped">
+                 <tbody>
+                   <% if @users.any? %>
+                    <%= render @users %>
```
Comment (nguyenvandatb) [inline app/views/users/_show_follow.html.erb]: space

File: app/views/users/_show_follow.html.erb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                   <% else %>
+                     <div class="alert alert-warning">
+                       <%= t "not_found_users" %>
+                     </div>
+                   <% end %>
+                 </tbody>
```
Comment (nguyenvandatb) [inline app/views/users/_show_follow.html.erb]: remove space

**Comment by nguyenvandatb** [comment]
OK

File: app/views/users/_show_follow.html.erb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      </div>
+      <div class="col-md-9 col-xs-12">
+        <div class="nav-tabs-custom">
+          <div class="feed_title">
+            <h2><%= t "followers" %></h2>
+          </div>
+          <div class="tab-content" id="show_follow">
+            <div class="table-responsive">
+               <table class="table table-striped">
+                  <tbody>
```
Comment (hungnh103) [inline app/views/users/_show_follow.html.erb]: indent

**Pull request merged**